ansible/roles_ocp_workloads/ocp4_workload_integreatly/tasks/post_workload.yml

```diff
@@ -2,6 +2,43 @@
 # Implement your Post Workload deployment tasks here
 # --------------------------------------------------
 
+# Create dedicated admin user
+
+- name: Get current htpasswd secret
+  shell: oc get secret htpasswd-secret -n openshift-config -o json | jq -j '.data.htpasswd' | base64 --decode > /tmp/htpasswd
+
+- name: Backup htpasswd
+  shell: cp /tmp/htpasswd /tmp/htpasswd.backup
+
+- name: Create dedicated admin user
+  shell: htpasswd -B -b /tmp/htpasswd {{ ocp4_workload_integreatly_dedicated_admin_username }} {{ ocp4_workload_integreatly_dedicated_admin_user_password }}
+
+- name: Ensure htpasswd Secret is absent
+  k8s:
+    state: absent
+    api_version: v1
+    kind: Secret
+    name: htpasswd-secret
+    namespace: openshift-config
+  retries: 3
+  delay: 5
```

Comment:
@dimitraz Can we cover the update with the `until` block in this PR?

Reply:
Resolved in https://github.com/redhat-cop/agnosticd/pull/1853